gulpfile.js

```diff
@@ -65,6 +66,26 @@ var buildConfig = {
       vendor: {
         src: 'source/frontend/vendor/js/development/vendor.js',
```

Comment:
src here is pointing to vendor.js. Do we need to change this back to all files in the development folder?